README.md

```diff
@@ -426,6 +426,38 @@ Using a file for a query is also supported with optional parameters to use if th
 const result = await sql.file('query.sql', ['Murray', 68])
 ```
 
+### Rows as Streams
+
+Postgres.js supports [`copy ...`](https://www.postgresql.org/docs/14/sql-copy.html) queries, which are exposed as [Node.js streams](https://nodejs.org/dist/latest-v16.x/docs/api/stream.html).
+
+> **NOTE** This is a low-level API which does not provide any type safety. To make this work, you must match your [`copy query` parameters](https://www.postgresql.org/docs/14/sql-copy.html) correctly to your [Node.js stream read or write](https://nodejs.org/dist/latest-v16.x/docs/api/stream.html) code. Ensure [Node.js stream backpressure](https://nodejs.org/en/docs/guides/backpressuring-in-streams/) is handled correctly to avoid memory exhaustion.
```

Comment:
It's tempting to mention the current default postgres COPY parameters here (i.e. tsv -- tabs for cells, new lines for rows), however, I think it's more useful to send readers to the postgres docs which cover this and more. 

Thoughts?

Reply:
Yeah, I agree with you - The PostgreSQL docs are great, but it can sometimes be hard to find what you're looking for if you aren't familiar with them. The link is great, but perhaps change `14` to `current` :)